x/ibc/07-tendermint/types/msgs.go

```diff
@@ -36,6 +36,7 @@ type MsgCreateClient struct {
 	TrustingPeriod  time.Duration   `json:"trusting_period" yaml:"trusting_period"`
 	UnbondingPeriod time.Duration   `json:"unbonding_period" yaml:"unbonding_period"`
 	MaxClockDrift   time.Duration   `json:"max_clock_drift" yaml:"max_clock_drift"`
+	ProofSpecs      []string        `json:"proof_specs" yaml:"proof_specs"`
```

Comment:
missing validation of specs

Reply:
can you add the specs field to the Msg constructor function too?

Reply:
Can't add meaningful validation of specs yet since we're still using the tendermint simple proof op for multistores for now.

Once I get that merged (next PR), then I can add some validation logic here by checking that *if* specs is not nil, each string is prefixed by `ics23:`

Reply:
Added some simple validation for now, will become more robust in the next PR